src/main/deltachat/chatlist.js

```diff
@@ -25,64 +25,88 @@ module.exports = class DCChatList extends SplitOut {
     return chat
   }
 
-  onChatModified (chatId) {
+  async onChatModified (chatId) {
     // TODO: move event handling to store
-    const chat = this.getFullChatById(chatId)
+    const chat = await this.getFullChatById(chatId)
     this._controller.sendToRenderer('DD_EVENT_CHAT_MODIFIED', { chatId, chat })
   }
 
-  getChatListIds (listFlags, queryStr, queryContactId) {
+  async __chatListGetChatId (list, index) {
```

Comment:
why these double underscore? does it mean "very" private function?

Reply:
I guess so. but I also believe we should use typescript for such things in the future